config/images/images.yaml

```
@@ -164,6 +164,16 @@ images:
   destination: europe-docker.pkg.dev/gardener-project/releases/3rd/opentelemetry-operator/opentelemetry-operator
   tags:
   - v0.127.0
+- source: persesdev/perses
+  destination: europe-docker.pkg.dev/gardener-project/releases/3rd/persesdev/perses
+  tags:
+  - v0.50.3
+  - v0.51.0
+- source: persesdev/perses-operator
+  destination: europe-docker.pkg.dev/gardener-project/releases/3rd/persesdev/perses-operator
+  tags:
+  - v0.1.10
+  - v0.2.0
```

gardener/gardener@master is at v0.1.12, it is also not updated to v0.2.0 with https://github.com/gardener/gardener/pull/12371/files#diff-d880f5ed5b194cea49bceb3f252990d7a8476de0027561917fca0452ebd2e150R404, so maybe you want ?

```suggestion
  - v0.1.12
```

---

Understood. Then I would prefer to also replicate the v0.1.12 image. I'll add it back so that we have 0.1.10, 0.1.12, and 0.2.0 for the operator.